rll/test-script.sh

```diff
@@ -0,0 +1,8 @@
+#! /bin/bash -ex
```

Comment:
Setting of `-x` already done on line 4.  If also doing `-e` line 4 should have that as well.